cypress/integration/submitInvalidContactForm.spec.js

```diff
@@ -0,0 +1,74 @@
+describe('submitInvalidContactForm',function () {
+    const operations = [
+        {
+            // Estonian
+            contactPage: 'https://ut2019.voog.com/contact-1',
+            result: 'Vigane sisend',
+            language: "Estonian"
+
+        },
+        {
+            // English
+            contactPage: 'https://ut2019.voog.com/contact',
+            result: "Invalid input",
+            language: "English"
+        }
+    ];
+
+    const testing = [
```

Comment:
We really like how you extracted the tests into a separate array so that more tests can easily be added with just additional array elements. However currently they are reported as a single test. You could consider looping over the it(); to report each test separately.